pkg/controller/argocd/statefulset.go

```diff
@@ -65,7 +83,7 @@ func newStatefulSetWithSuffix(suffix string, component string, cr *argoprojv1a1.
 }
 
 func (r *ReconcileArgoCD) reconcileRedisStatefulSet(cr *argoprojv1a1.ArgoCD) error {
-	ss := newStatefulSetWithSuffix("redis-ha-server", "redis", cr)
+	ss := newStatefulSetWithSuffix("redis-ha", "redis", cr)
```

Comment:
I am curious why this renaming.

Reply:
it was updated for consistency. For the instance name, it was declared as `redis-ha-server` while the value for labels was provided as `redis-ha`. 
Also, the service account name was hardcoded to `argocd-redis-ha` which did not follow the same convention as other resources.